src/zino/zino.py

```diff
@@ -45,6 +45,9 @@ def init_event_loop(args: argparse.Namespace, loop: Optional[AbstractEventLoop]
         trap_receiver.add_community("public")
         trap_receiver.add_community("secret")
 
+        # ensure observers are loaded
+        from zino.trapobservers import ignored_traps  # noqa
```

Comment:
Why do we have to do this here? And will we have to import all trap observers like this here?

Reply:
It doesn't have to happen here, it can be a normal module-level import.  But yes.  Every trap observer we want to include in the daemon would have to be imported explicitly.  If not, we also have to make a mechanism that finds all the modules and loads them - and for some things I like to just defer to the Zen of Python: _"Explicit is better than implicit"_ :grin: